data-collection/deploy/module-rds-multitenant.yaml

```diff
@@ -0,0 +1,677 @@
+                      s3_key = f"{metrics_s3_prefix}/payer_id={payer_id}/account_id={account_id}/region={region}/year={year}/month={month}/day={day}/hour={hour}/metrics.parquet"
+
+                      print(f"Writing metrics to S3 key: {s3_key}")
+                      print(f"Total number of metrics: {len(grouped_metrics)}")
+
+                      # Convert to Arrow table
+                      table = pa.Table.from_pylist(grouped_metrics)
+
+                      # Write to Parquet format
+                      buf = BytesIO()
+                      pq.write_table(table, buf)
+                      buf.seek(0)
+
+                      # Upload to S3
+                      s3_client.put_object(
+                          Bucket=os.environ['METRICS_BUCKET'],
+                          Key=s3_key,
+                          Body=buf.getvalue()
+                      )
+                      print(f"Successfully wrote metrics to {s3_key}")
+      Environment:
+        Variables:
+          METRICS_BUCKET: !Ref DestinationBucket
+          METRICS_PERIOD_IN_SECONDS: '3600'
+          METRICS_S3_PREFIX: 'rds-multitenant'
+          ROLENAME: !Ref MultiAccountRoleName
+
+  ModuleRefreshSchedule:
+    Type: 'AWS::Scheduler::Schedule'
+    Properties:
+      Description: !Sub 'Scheduler for the ODC RDS Multitenant module'
+      Name: !Sub '${ResourcePrefix}RDSMultitenant-RefreshSchedule'
+      ScheduleExpression: !Ref Schedule
+      State: ENABLED
+      FlexibleTimeWindow:
+        MaximumWindowInMinutes: 30
+        Mode: 'FLEXIBLE'
+      Target:
+        Arn: !GetAtt ModuleStepFunction.Arn
+        RoleArn: !Ref SchedulerExecutionRoleARN
+        Input: !Sub '{"module_lambda":"${RDSPerformanceInsightsFnHourly.Arn}","crawlers": ["${ResourcePrefix}PerformanceInsightsRDSCrawler", "${ResourcePrefix}PerformanceInsightsRDSCrawlerHourly"]}'
+
+  GlueDatabase:
+    Type: AWS::Glue::Database
+    Properties:
+      CatalogId: !Ref AWS::AccountId
+      DatabaseInput:
+        Name: rds_performance_insights_db
+        Description: Database for RDS Performance Insights data
+
+
+
+  GlueServiceRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub "${ResourcePrefix}RDSMultitenant-Glue-Role"
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: glue.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSGlueServiceRole"
+      Policies:
+        - PolicyName: S3Access
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:GetObject
+                  - s3:PutObject
+                Resource: !Sub "${DestinationBucketARN}/*"
+
+  PerformanceInsightsRDSCrawler:
```

Comment:
the patttern is to run crawler in step function that we have already. please check other modules